BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/location/projection/SphericalMercatorProjectionTest.java

```diff
@@ -50,4 +51,14 @@ public void conversion() throws Exception {
         assertEquals(expectedLongitude, actualLongitude, 0.00001);
     }
 
+    @Test
+    public void conversionWithAltitude() throws Exception {
+        Location expectedLocation = LocationTest.BERLIN;
+        expectedLocation.setElevation(2);
```

Comment:
Why is the elevation specified? It shouldn't be relevant.

Reply:
Elevation is used for 3D multilateration